pkg/model/utxo/treasury_output.go

```diff
@@ -34,6 +35,41 @@ type TreasuryOutput struct {
 	Spent bool
 }
 
+type jsonTreasuryOutput struct {
```

Comment:
what is this used for? There is still a `treasuryResponse` in the coreapi pkg with the same JSON structure

Reply:
It is used to print the snapshot header as JSON in `printFullSnapshotHeaderInfo` in `pkg/toolset/snap_merge.go`

Reply:
Then maybe remove `treasuryResponse` and use this instead